### PR TITLE
Add Empty armor icons (minecraft), EMI, Roundalib, AuthMe and Fabric creative menu tabs

### PR DESCRIPTION
**Pull request by xllifi**

### CODE REVIEW

**Comment by xllifi** [comment]
Roundalib is Inventory Management's embedded lib. Changes following buttons:

**Comment by GabrielDja** [comment]
**Thank you for helping the project,**
The changes will be applied in the next update.
(Some textures will be reworked)

File: assets/roundalib/textures/gui.png
Comment (GabrielDja) [inline assets/roundalib/textures/gui.png:1]: Which project does this source come from?
**Can you send me a link to this project?**
Thanks :D

Comment (xllifi) [reply assets/roundalib/textures/gui.png:1]: Inventory Management's embedded library
Path: `inventorymanagement.jar\META-INF\jars\roundalib-0.2.1+1.19.jar\assets\roundalib\textures\`

Comment (GabrielDja) [reply assets/roundalib/textures/gui.png:1]: Okay, **thanks** 😄

**Comment by xllifi** [comment]
What about Fabric creative menu tabs and AuthMe?
Reference texture for Fabric creative menu tabs is located at: ```1.19.2-Fabric-API-0.76.0-1.19.2.jar/META-INF/jars/fabric-item-groups-v0-0.76.0.jar/assets/fabric/textures/gui/creative_buttons.png``` for Fabric API 1.19.2 0.76.0

**Comment by GabrielDja** [comment]
> Qu'en est-il des onglets du menu créatif Fabric et d'AuthMe ? La texture de référence pour les onglets du menu créatif Fabric se trouve à : `1.19.2-Fabric-API-0.76.0-1.19.2.jar/META-INF/jars/fabric-item-groups-v0-0.76.0.jar/assets/fabric/textures/gui/creative_buttons.png`pour l'API Fabric 1.19.2 0.76.0

**Auth Me support added for version 2.0 (Big Update)**
**And for Fabric too.**

**Pull request closed without merging**